www/deprecation_policy.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: contribute
+title: Apache Cordova Deprecation Policy
+---
+
+# Deprecation and Archiving Policy
+
+Apache Cordova has been around quite some time, having been started back in August 2008. As it consists of many individual "parts", some of those stopped being useful for any number of reasons. This "Deprecation and Archiving Policy" describes why, when and how projects can be deprecated and archived - and what that actually means.
+
+## Deprecation Policy
+
+When a piece of Apache Cordova stops being useful or necessary we vote on deprecating it on our [dev mailing list](/contact/).
+
+### Reasons for Deprecation
+
+Common reasons for deprecation of a piece of Apache Cordova are:
```

Comment:
I would add the word "Some" at the beginning, to cover for the case where we want to deprecate for any other reasons.

Reply:
```suggestion
Some common reasons for deprecation of a piece of Apache Cordova are:
```